pom.xml

```diff
@@ -256,6 +256,33 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.sonatype.ossindex.maven</groupId>
+                <artifactId>ossindex-maven-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <id>audit-dependencies-critical</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>audit</goal>
+                        </goals>
+                        <!-- configuration for mvn validate -->
+                        <configuration>
+                            <!-- if CVSS >= 9.0 (critical) then ERROR else WARN -->
+                            <fail>true</fail>
+                            <cvssScoreThreshold>9.0</cvssScoreThreshold>
+                        </configuration>
+                    </execution>
+                </executions>
+                <!-- configuration for mvn ossindex:audit -->
+                <configuration>
+                    <!-- if CVSS >= 7.0 (high or critical) then ERROR else WARN -->
+                    <fail>true</fail>
+                    <cvssScoreThreshold>7.0</cvssScoreThreshold>
+                </configuration>
```

Comment:
I think for now you'll need to add this for the jackson dependency as we can't fix that one easily. I believe we are stuck on this version because of the Elastic search version we are using.
```
<excludeCoordinates>
    <exclude>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-databind</artifactId>
        <version>2.9.7</version>
    </exclude>
</excludeCoordinates>
```

Reply:
@daiLlew I think you are probably right. There is another ticket in the backlog to address the jackson-databind vulnerability. Can we leave this erroring for now and we will add the exclude if needed once we get to that story?

Reply:
I've been overruled so I don't really have a choice.